imagepipeline/src/main/java/com/facebook/imagepipeline/core/ProducerFactory.java

```diff
@@ -120,7 +120,7 @@ public ProducerFactory(
       ByteArrayPool byteArrayPool,
       ImageDecoder imageDecoder,
       ProgressiveJpegConfig progressiveJpegConfig,
-      boolean downsampleEnabled,
+      DownsampleMode downsampleMode,
```

Comment:
This is a breaking change (e.g. for callers or subclasses of `ProducerFactory`).

I think we need to add a constructor overload that continues accepting a `boolean isDownsampleEnabled` in this argument position.

Reply:
added